data_source_alks_session.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"github.com/Cox-Automotive/alks-go"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAlksSession() *schema.Resource {
```

Comment:
@stephenleemcfarland Do you think `alks_session` is the right name for this data source? Would `alks_credentials` or `alks_session_credentials` be more clear?

Reply:
maybe `aws_session_credentials` cause that's what it really is? I don't feel strongly about this. I think your current name is reasonable already